input/definitions/Resource/CodeSystem-negation-reason-example.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Actually, remove this whole CodeSystem, it is not necessary, we only need to create a ValueSet example with a few of the codes from the negationReasonCodes value set in QICore